### ISSUE TITLE
[Bug]: Explore Tab Toast Appears Repeatedly when Location is off

### ISSUE THREAD

**Issue by shashankiitbhu** [commons-app/apps-android-commons#5551]
### Summary

Explore Tab Toast Appears Repeatedly when Location is off, which results in multiple same toasts appearing over and over again.

Additionally, Ideally this Toast Should Appear when clicked on map

### Steps to reproduce

1. Open the app
2. Go to Explore Tab
3. Tab on Uploaded Via Mobile

### Expected behaviour

Single Toast

### Actual behaviour

Multiple Toasts

### Device name

Xaomi 11 Lite NE

### Android version

Android 13

### Commons app version

4.2.1-debug-main

### Device logs

2024-02-15 19:46:24.878  1867-4055  NotificationService     system_server                        E  Package has already queued 5 toasts. Not showing more. Package=fr.free.nrw.commons.beta

### Would you like to work on the issue?

Yes

**Comment by ShashwatKedia** [comment]
The toast appearing when clicked on 'Uploaded via mobile' seems to be a duplicate of #5549. And about the Toast appearing multiple times, it will be resolved by #5494.

**Comment by shashankiitbhu** [comment]
@ShashwatKedia Yes sure, I think fixing this issue - The toast appearing when clicked on 'Uploaded via mobile' , together with #5549 will be better
